tests/Functional/Cases/Parsing/NamespaceRegistrationTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace TYPO3Fluid\Fluid\Tests\Functional\Cases\Parsing;
+
+use TYPO3Fluid\Fluid\Tests\Functional\BaseFunctionalTestCase;
+
+/**
+ * Class NamespaceRegistrationTest
+ */
+class NamespaceRegistrationTest extends BaseFunctionalTestCase
+{
+
+    /**
+     * @var array
+     */
+    protected $variables = [];
+
+    /**
+     * @return array
+     */
+    public function getTemplateCodeFixturesAndExpectations()
+    {
+        return [
```

Comment:
I know it's just a revert but no matter how long I look at this test, I simply don't get the intention. Especially that the output which is expected and which is not expected is basically the same.